infra/terraform/test-org/ci-comment-build-trigger-function/function.tf

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
nit: maybe we can also add a comment that this is not used any more but maybe open an issue to delete it from repo at somepoint

Reply:
SG - Added